mysql-test/suite/sys_vars/t/path_resolution.test

```diff
@@ -0,0 +1,4718 @@
+--echo #
+--echo # Path resolution
```

Comment:
Include the MDEV before the Path Resolution comment.

This test is actually testing functionality so shouldn't be part of the sys_vars suite. As is on test in main is acceptable. Include a "echo # End of 11.8 tests" at end. This make merging later easier.

Reply:
Changed